src/applications/personalization/dashboard/tests/components/health-care/AppointmentsCard.unit.spec.jsx

```diff
@@ -15,43 +15,21 @@ describe('<AppointmentsCard />', () => {
     },
   };
 
-  it('should render without appointments', () => {
-    const start = moment.parseZone();
-    const startFormatted = start.format('dddd, MMMM Do, YYYY');
-    const timeZone = getAppointmentTimezone();
-    const tree = renderWithStoreAndRouter(<AppointmentsCard />, {
-      initialState,
-    });
-
-    tree.getByTestId('health-care-appointments-card');
-    tree.getByText('Next appointment');
-    tree.getByText('Schedule and manage your appointments');
-    expect(
-      tree.getByRole('link', {
-        name: /schedule and manage your appointments/i,
-        value: {
-          text: '/my-health/appointments',
-        },
-      }),
-    ).to.exist;
-    tree.getByText(startFormatted);
-    tree.getByText(`Time: ${start.format('h:mm a')} ${timeZone.abbreviation}`);
-  });
-
```

Comment:
I removed it because in the case of a user have no appointments, the AppointmentCard component shouldn't be visible and they'll see the No Upcoming Appointments Text. this is handled in [HealthCareContent tests](https://github.com/department-of-veterans-affairs/vets-website/blob/My-VA/allison/62294/update-moment-to-datefns/src/applications/personalization/dashboard/tests/components/health-care/HealthCareContent.unit.spec.jsx#L70-L77)